content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -405,6 +410,113 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry environment
+
+In OpenTelemetry environments, you can use the [OpenTelemetry Collector][16] to collect and batch telemetry data (like metrics, traces, and logs) by leveraging the [Datadog Exporter][17] to send the data to Datadog without the Datadog Agent.
+
+Optionally, you can send telemetry data from [applications instrumented with the OpenTelemetry SDKs with the Datadog Agent][20], which collects the data for Datadog.
+
+To set up unified service tagging in an OpenTelemetry environment:
+
+1. Set the required attributes to ensure proper tagging for the OpenTelemetry component you are using.
+
+   For example, with the Python OpenTelemetry SDK:
+
+   ```python
+   from opentelemetry.sdk.resources import Resource
+   from opentelemetry.sdk.trace import TracerProvider
+
+   resource = Resource(attributes={
+      "service.name": "<SERVICE>",
+      "deployment.environment": "<ENV>",
+      "service.version": "<VERSION>"
+    })
+    tracer_provider = TracerProvider(resource=resource)
+    ```
+
+    For example, with the OpenTelemetry Collector:
+
+    ```yaml
+    receivers:
+    otlp:
+      protocols:
+        grpc:
+        http:
+
+    processors:
+      batch:
+
+    exporters:
+      datadog:
+        api:
+          key: "<DATADOG_API_KEY>"
+        metrics:
+          attributes:
+            service.name: "<SERVICE>"
+            deployment.environment: "<ENV>"
+            service.version: "<VERSION>"
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [datadog]
+    ```
+
+   
+1. To [correlate OpenTelemetry language SDK logs and traces][18], translate the OpenTelemetry `TraceId` and `SpanId` properties (a 128-bit unsigned integer and 64-bit unsigned integer represented as a 32-character and 16-character hexadecimal string) into the Datadog format and add them as attributes in your logs. 
```

Comment:
We have a dedicated page for this, right? I don't think we need to repeat it here